kong/init.lua

```diff
@@ -695,6 +695,11 @@ function Kong.init()
     if config.cluster_rpc then
       kong.rpc = require("kong.clustering.rpc.manager").new(config, kong.node.get_id())
 
+      if config.cluster_incremental_sync then
+        kong.sync = require("kong.clustering.services.sync").new(db, is_control_plane(config))
+        kong.sync:init(kong.rpc)
```

Comment:
`kong.sync` is a little generic. Can we do better?

Reply:
Do you have any suggestion for it?

Reply:
maybe `kong.config_sync`? I believe we're leaving `incremental` out on purpose here because this sync can be `full`

Reply:
KAG-5590, let's do it later.